gui/accelmap.py

```diff
@@ -291,14 +342,14 @@ def _accel_edited_cb(self, cell, path, newname):
 
     def _accel_editing_started_cb(self, cell, editable, treepath):
         """Begin editing by showing a key capture dialog"""
-        store = self._store
-        it = store.get_iter(treepath)
-        accel_path = store.get_value(it, self._PATH_COLUMN)
+        it = self._filter.get_iter(treepath)
+        accel_path = self._filter.get_value(it, self._PATH_COLUMN)
         accel_label = self._accel_labels[accel_path]
         action_label = self._action_labels[accel_path]
 
         editable.set_sensitive(False)
         dialog = Gtk.Dialog()
+        dialog.set_modal(True)
```

Comment:
Why is it necessary for this dialog to be modal?

Reply:
When clicking multiple times, a number of dialogs were appearing at the same time, so I thought this would be more correct behavior, and fixed it.

Reply:
Fair enough, it does make sense in this case.